content/blog/2022-05-16-may-22-heartbeat.md

```diff
@@ -0,0 +1,393 @@
+[**Isabel Bär,**](https://www.linkedin.com/in/isabel-b%C3%A4r-a89705213/)
+[**Alexander Kniesz,**](https://www.linkedin.com/in/alexander-kniesz-656256197/)
+and [**Michael Plöd**](https://www.linkedin.com/in/michael-ploed/) of
+[**INNOQ**](https://www.innoq.com/en/) (a software development, strategy, and
+technology consultancy) created
+[this](https://ml-ops.org/content/mlops-principles) very thorough resource on
+MLOps, going through all the principles and "iterative-incremental" steps of the
+process (there's an iterative pattern here 😉). The authors cover Automation,
+Continuous X (hello CML and TPI), Versioning (hello DVC!), Experiments Tracking
+(noted DVC here because indeed DVC does experiment tracking too!), Testing,
+Monitoring, the "ML Test Score" System, Reproducibility, Modularity, ML-based
+Software Delivery Metrics, and MLOps Principles and Best Practices. Definitely a
+good resource for for MLOps and filled with more resources as well.
+
+![INNOQ MLOps Guide](/uploads/images/2022-05-16/innoq.jpeg '=800') _INNOQ MLOps
+Guide ([Source link](https://ml-ops.org/content/mlops-principles))_
+
+Also interesting from INNOQ is their
+[Artist-in-residence program](https://www.innoq.com/en/artists/) created because
+they "believe in the conscious reflection between technology and society" and
+feel art is well suited for this refection. See the work below by Studio Waltz
+Binaire based on the question: What traces do we leave behind with technology?
+
+https://media.giphy.com/media/NxdrJ6a4IQKyW5gGjL/giphy.gif
+
+([Source link](https://www.innoq.com/en/artists/))
+
+## Laszlo Sragner: LinkedIn discussion on Code Quality
+
+[**Laszlo Sragner**](https://www.linkedin.com/in/laszlosragner/?trk=public_post-embed_share-update_actor-text&originalSubdomain=uk)
+a frequent contributor to the MLOps Community in general, often driving
+discussions and helping others in the
+[MLOps Community Slack channel,](https://mlops-community.slack.com/join/shared_invite/zt-178s99cyv-Q~whRpqbhgMTBrOcbjnDIQ#/shared-invite/email)
+posed an interesting point about code quality on LinkedIn. Join the discussion
+and weigh in at this post:
+
+<iframe src="https://www.linkedin.com/embed/feed/update/urn:li:share:6931541880090324992" height="800" width="504" frameborder="0" allowfullscreen="" title="Embedded post"></iframe>
+
+# Company News
+
+## ICYMI: We released TPI! 🎉
+
+On April 27th we released the latest offering to our tool ecosystem.
+
+https://media.giphy.com/media/ut7lqhIfOscbjuU6YQ/giphy.gif
+
+[Terraform Provider Iterative (TPI)](https://tpi.cml.dev) is a Terraform plugin
+built with machine learning in mind. Full lifecycle management of computing
+resources (including GPUs and respawning spot instances) from several cloud
+vendors (AWS, Azure, GCP, K8s)... without needing to be a cloud expert.
+
+- **Lower cost with spot recovery:** transparent data checkpoint/restore &
+  auto-respawning of low-cost spot/preemptible instances
```

Comment:
```suggestion
  auto-respawning of low-cost spot/pre-emptible instances
```

Reply:
There's apparently not a hyphen in that word.  https://www.google.com/search?q=preemtible&oq=preemtible&aqs=chrome.0.69i59j0i10l9.5603j0j7&sourceid=chrome&ie=UTF-8

Weird to me too!

Reply:
Huh! English is weird. 😅